packages/account-v2/tailwind.config.js

```diff
@@ -0,0 +1,9 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+    content: ['./src/**/*.{js,jsx,ts,tsx}'],
+    important: '.quill-design',
+    presets: [require('@deriv/quill-design/quill-tailwind/tailwind.config.cjs')],
+    corePlugins: {
+        preflight: false,
```

Comment:
`preflight` set to `false` disables the use of default stylings by tailwind.
<img width="813" alt="Screenshot 2024-01-04 at 4 18 25 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/8c15f614-e768-4f78-93e5-39f0048190ae">